features/pages/location_page.rb

```diff
@@ -0,0 +1,10 @@
+module Pages
+  class Location < SitePrism::Page
+    set_url '/locations/{id}'
```

Comment:
It is a conscious decision yes. It was defined as a `resources` in the routes file to facilitate exactly this.